Makefile

```diff
@@ -1,13 +1,22 @@
-default: clean static-checks test build post-build
+VERSION ?= dev
```

Comment:
comment to this variable would not harm in context of their behaviour perhaps

Reply:
@MalloZup good call! I added some comments for all the Makefile variables.